app/models/user.rb

```diff
@@ -277,6 +277,20 @@ def unmoderated_in_period(start_time, end_time)
         .distinct
   end
 
+  def moderator_queue
```

Comment:
Does this make more sense in a controller, maybe? Or, could you explain your reasoning in a comment in the code? Thanks!

Reply:
@jywarren, I have added this in the controller. Thanks!!